phpunit/functional/Glpi/Form/QuestionType/QuestionTypeAssigneeTest.php

```diff
@@ -105,61 +109,284 @@ public static function assigneeAnswerIsDisplayedInTicketDescriptionProvider(): i
         ];
     }
 
-    #[DataProvider("assigneeAnswerIsDisplayedInTicketDescriptionProvider")]
-    public function testAssigneeAnswerIsDisplayedInTicketDescription(
-        array $answer,
-        string $expected,
-        bool $is_multiple
-    ): void {
+    public static function invalidActorsProvider(): iterable
+    {
+        yield 'invalid user' => [
+            [User::getForeignKeyField() . "-999999"],
+            'expected_exception' => \Exception::class,
+            'expected_message' => "Invalid actor ID: 999999",
+        ];
+
+        yield 'invalid group' => [
+            [Group::getForeignKeyField() . "-999999"],
+            'expected_exception' => \Exception::class,
+            'expected_message' => "Invalid actor ID: 999999",
+        ];
+
+        yield 'invalid supplier' => [
+            [Supplier::getForeignKeyField() . "-999999"],
+            'expected_exception' => \Exception::class,
+            'expected_message' => "Invalid actor ID: 999999",
+        ];
+
+        yield 'invalid user and group' => [
+            [User::getForeignKeyField() . "-999999", Group::getForeignKeyField() . "-999999"],
+            'expected_exception' => \Exception::class,
+            'expected_message' => "Invalid actor ID: 999999",
+        ];
+
+        yield 'valid user and invalid group' => [
+            [
+                User::getForeignKeyField() . "-" . getItemByTypeName(User::class, "glpi", true),
+                Group::getForeignKeyField() . "-999999"
+            ],
+            'expected_exception' => \Exception::class,
+            'expected_message' => "Invalid actor ID: 999999",
+        ];
+
+        yield 'multiple valid actors for single actors question' => [
+            [
+                User::getForeignKeyField() . "-" . getItemByTypeName(User::class, "glpi", true),
+                Group::getForeignKeyField() . "-" . getItemByTypeName(Group::class, "_test_group_1", true),
+            ],
+            'expected_exception' => \Exception::class,
+            'expected_message' => "Multiple actors are not allowed",
+        ];
+    }
+
+    public static function validActorsProvider(): iterable
+    {
+        $glpi_id = getItemByTypeName(User::class, "glpi", true);
+        $tech_id = getItemByTypeName(User::class, "tech", true);
+        $test_group_1_id = getItemByTypeName(Group::class, "_test_group_1", true);
+        $test_group_2_id = getItemByTypeName(Group::class, "_test_group_2", true);
+        $supplier_01_id = getItemByTypeName(Supplier::class, "_suplier01_name", true);
+        $supplier_02_id = getItemByTypeName(Supplier::class, "_suplier02_name", true);
+
+        yield 'valid user' => [
+            [User::getForeignKeyField() . "-$glpi_id"],
+            [
+                [
+                    'itemtype' => User::class,
+                    'items_id' => $glpi_id,
+                ]
+            ],
+            false
```

Comment:
You should use parameter names as keys in your test providers. It helps to understand them when code is reviewed.

```suggestion
            'answer' => [User::getForeignKeyField() . "-$glpi_id"],
            'expected' => [
                [
                    'itemtype' => User::class,
                    'items_id' => $glpi_id,
                ]
            ],
            'allow_multiple_actors' => false
```